.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ["bug"]
+projects: ["livechat/7"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See an error
+    validations:
+      required: false
```

would set that as required also
```suggestion
      required: true
```